homeassistant/components/verisure.py

```diff
@@ -116,15 +121,24 @@ def setup(hass, config):
 
 def reconnect():
     """ Reconnect to verisure mypages. """
+    if not LOCK.acquire(blocking=False):
+        return
+    global PASSWORD_RETRIES
     try:
-        time.sleep(1)
+        time.sleep(10)
```

Comment:
1 thread is responsible for handling all updates to sensor entities. You would still need a lock since there are multiple verisure platforms which could still be active at the same time.